examples/php-features-test/proto/test/v1/test.proto

```
@@ -0,0 +1,152 @@
+  TEST_ENUM_ACTIVE = 1;
+  TEST_ENUM_INACTIVE = 2;
+  TEST_ENUM_PENDING = 3;
+}
+
+// Service definition for gRPC testing
+service TestService {
+  // Unary RPC
+  rpc GetTest(GetTestRequest) returns (GetTestResponse);
+
+  // Server streaming RPC
+  rpc ListTests(ListTestsRequest) returns (stream TestMessage);
```

Buf (1.54.0)

"test.v1.TestMessage" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)